client/dom/radiobutton.ts

```diff
@@ -62,11 +62,16 @@ export function make_radios(opts: RadioButtonOpts): RadioApi {
 		.style('display', 'block')
 		.data(opts.options, (d: any) => d?.value)
 
+	/** Fix to allow the display to be customizable but show buttons
+	 * in a single line if 3 or less options. Otherwise display vertically. */
+	const displayMode =
+		opts.styles && 'display' in opts.styles ? opts.styles.display : opts.options.length <= 3 ? 'inline-block' : 'block'
```

Comment:
not just 3 or less options, but if the option labels are too long then inline-block is not appropriate. can get total number of characters from option labels. if under a cutoff, allow inline. e.g. this allows even 4 options with short labels to inline